src/widgets/speakers/ui/speakers.test.tsx

```diff
@@ -33,7 +33,9 @@ describe('Speakers', () => {
   });
 
   it('renders the image correctly', () => {
-    const imageElement = screen.getByAltText('speakers-wanted');
+    const imageElement = screen.getByAltText(
+      'Cartoon sloth wearing a yellow shirt, gesturing with a speech bubble',
+    );
 
     expect(imageElement).toBeInTheDocument();
   });
```

Comment:
Maybe we can add a `data-testid `attribute to the picture and do a search for the item through that attribute. And the `alt` and maybe `src` attributes should be checked below?

Reply:
done